atlasdb-commons/src/main/java/com/palantir/util/IndexEncodingUtils.java

```diff
@@ -111,8 +110,7 @@ public static <K extends DeterministicHashable, V, R> Map<K, R> decode(
         return keyToValue;
     }
 
-    @VisibleForTesting
-    static <K extends DeterministicHashable> KeyListChecksum computeChecksum(
+    public static <K extends DeterministicHashable> KeyListChecksum computeChecksum(
```

Comment:
We could get away with not making this public, but it makes the test setup considerably uglier (we would have to invoke `encode` and extract the checksum from there.